skins/neowx-material/skin.conf

```
@@ -380,7 +398,7 @@
 # This section describes all templates and their generated files
 #
 [CheetahGenerator]
-    search_list_extensions = user.historygenerator.MyXSearch
+    search_list_extensions = user.historygenerator.MyXSearch, user.openmeteo.Forecast
 
```

It's now enabled by default
please 
```suggestion
    search_list_extensions = user.historygenerator.MyXSearch
    # to enable forecast uncomment next line and comment line above....
    # search_list_extensions = user.historygenerator.MyXSearch, user.openmeteo.Forecast
```